.docker/ci/Dockerfile

```diff
@@ -1,75 +1,42 @@
-
-# setup sources.list
-RUN echo "deb http://packages.ros.org/ros/ubuntu bionic main" > /etc/apt/sources.list.d/ros-latest.list
-
-# hack to fix issue with tzdata install
-# https://stackoverflow.com/questions/8671308/non-interactive-method-for-dpkg-reconfigure-tzdata
-ENV DEBIAN_FRONTEND=noninteractive
-ENV DEBCONF_NONINTERACTIVE_SEEN=true
-
-# install bootstrap tools
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    python-rosdep \
-    python-rosinstall \
-    python-vcstools \
-    && rm -rf /var/lib/apt/lists/*
-
-# setup environment
-ENV LANG C.UTF-8
-ENV LC_ALL C.UTF-8
-
-# bootstrap rosdep
-RUN rosdep init \
-    && rosdep update
-
-ENV ROS_DISTRO melodic
 ENV TERM xterm
 
 # Setup catkin workspace
 ENV CATKIN_WS=/root/ws_moveit
 RUN mkdir -p $CATKIN_WS/src
 WORKDIR $CATKIN_WS/src
 
-# Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
-RUN wstool init . && \
-    # Download moveit source so that we can get necessary dependencies
-    wstool merge https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
-    wstool update && \
-    # Update apt because previous images clear this cache
-    apt-get -qq update && \
-    # Install some base dependencies
-    apt-get -qq install -y \
-        # Some source builds require a package.xml be downloaded via wget from an external location
-        wget \
-        # Required for rosdep command
-        sudo \
-        # Preferred build tool
-        python-catkin-tools && \
-    # Download all dependencies of MoveIt!
-    rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false && \
-    # Remove the source code from this container. TODO: in the future we may want to keep this here for further optimization of later containers
-    cd .. && \
-    rm -rf src/ && \
-    # Clear apt-cache to reduce image size
-    rm -rf /var/lib/apt/lists/*
+# Download moveit source so that we can get necessary dependencies
+RUN wstool init .
+RUN wstool merge https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall
+RUN wstool rm moveit_visual_tools  # graph_msgs is not yet available
+RUN wstool update
+
+RUN apt-get -qq update
+
+# Install some base dependencies
+RUN apt-get -qq install -y \
+	# Some source builds require a package.xml be downloaded via wget from an external location
+	wget \
+	# Required for rosdep command
+	sudo \
+	# Preferred build tools
+	python-catkin-tools \
+	ccache
+        
+# Download all dependencies of MoveIt!
+RUN rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false
+
+# Remove the source code from this container. TODO: in the future we may want to keep this here for further optimization of later containers
+RUN cd .. && rm -rf src/
+
+# Clear apt-cache to reduce image size
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
this does not work as explained above

Reply:
It works perfectly as proven by [Travis](https://travis-ci.org/ros-planning/moveit/jobs/380463858#L1637) and locally.